include/sys/zfs_context.h

```diff
@@ -273,6 +273,7 @@ typedef struct kmutex {
 } kmutex_t;
 
 #define	MUTEX_DEFAULT	0
+#define	MUTEX_FSTRANS	MUTEX_DEFAULT
```

Comment:
Dear @behlendorf , please educate me a bit.... how does defining preprocessor variable MUTEX_FSTRANS to the same value as MUTEX_DEFAULT changes anything is the functioning?  It will still compile to the same value as far as I see it, or MUTEX_DEFAULT is redefined somewhere (not visible here) to be a different value?